### ISSUE TITLE
Training loss not dropping (Gaussian and GI

### ISSUE THREAD

**Issue by peterdn1** [AlexeyAB/darknet#4252]
I am trying out both Gaussian_yolo and GIOU
iou_normalizer=-0.5
iou_loss=giou
Started learning rate at 0.001 and avg loss went to -nan after about 1300 iterations and remained there.
Restarted:
Set the learning rate to 0.0001 and avg loss is not dropping (25000 iterations sitting at ~400-440

Training with yolov3 same dataset trained well. 

Can I use GIOU and Gaussian together?  If so any suggestions?

Thanks

**Comment by AlexeyAB** [comment]
> Can I use GIOU and Gaussian together?

GIoU isn't implemented in Gaussion-yolo yet.

> Set the learning rate to 0.0001 and avg loss is not dropping (25000 iterations sitting at ~400-440

This is the normal loss for Gaussion-yolo networks. Look at the mAP.

**Comment by peterdn1** [comment]
Ran the partially trained network with the avg loss of ~400 and results looked good, thank you.  

Can the loss calculation be adjusted the graphing function makes sense for Gaussian 

If you combine GIoU and Gaussian do you expect a linear improvement in mAP? (Gaussian mAP gain+GIoU gain)
